src/Agent.Sdk/ProcessInvoker.cs

```diff
@@ -805,6 +844,25 @@ private void NixKillProcessTree()
             }
         }
 
+        private void WriteProcessOomScoreAdj(int processId, int oomScoreAdj)
+        {
+            try
+            {
+                string procFilePath = $"/proc/{processId}/oom_score_adj";
+                // Linux by default mounts procfs at /proc, NOP on other platforms such as macOS.
+                if (File.Exists(procFilePath))
+                {
+                    File.WriteAllText(procFilePath, oomScoreAdj.ToString());
+                    Trace.Verbose($"Updated invoked process oom_score_adj (PID: {processId}) to {oomScoreAdj}.");
```

Comment:
Trace.Info(),  verbose won't shows up in release bits.
PS. change the trace message a little bit. :D

Reply:
Updated this